src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ITupleExpression.cs

```diff
@@ -186,6 +188,83 @@ static void Main()
 
         [CompilerTrait(CompilerFeature.IOperation)]
         [Fact, WorkItem(10856, "https://github.com/dotnet/roslyn/issues/10856")]
+        public void TupleExpression_ImplicitConversionsWithTypedExpression_02()
+        {
+            string source = @"
+using System;
+
+class C
+{
+    static void Main()
+    {
+        int a = 1;
+        int b = 2;
+        (long, long) t /*<bind>*/= (a, b)/*</bind>*/;
+        Console.WriteLine(t);
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableInitializerOperation (OperationKind.VariableInitializer, Type: null) (Syntax: '= (a, b)')
+  IConversionOperation (TryCast: False, Unchecked) (OperationKind.Conversion, Type: (System.Int64, System.Int64), IsImplicit) (Syntax: '(a, b)')
+    Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+    Operand: 
+      ITupleOperation (OperationKind.Tuple, Type: (System.Int64 a, System.Int64 b)) (Syntax: '(a, b)')
```

Comment:
Just to make sure that I'm understanding the conversion here, the actual thing being converted here is that the names of the tuple elements are being erased, correct? As both elements are still longs.

Reply:
Your understanding feels correct.